.github/workflows/docbuild-and-upload.yml

```diff
@@ -89,3 +89,10 @@ jobs:
         name: website
         path: web/build
         retention-days: 14
+
+    - name: Trigger web/doc preview
+      run: curl -X POST https://pandas.pydata.org/preview/submit/$RUN_ID/$PR_ID/
+      env:
+        RUN_ID: ${{ github.run_id }}
+        PR_ID: ${{ github.event.pull_request.number }}
+      if: github.event_name == 'pull_request'
```

Comment:
IIUC, this would run every time the PR is pushed to, so we might run out of disk space on the VM hosting our docs?

Reply:
Thanks for the review, and for finding the pending renamed from copying the assign job. I addressed them.

> IIUC, this would run every time the PR is pushed to, so we might run out of disk space on the VM hosting our docs?

Yes, you're correct. We overwrite the docs, so even if it's updated at every push, we only need one version per PR. But we'll eventually run out of space, since no purging is yet implemented. My idea is to get this working and see how it works for a bit, and if there are no surprises, and this is useful and convenient, I'll continue the pending things. There are some other things pending besides purging, like stop using my personal token, set up the service with systemd, improve the logging. Nothing big, but I think it makes more sense to start using the current MVP first, just in case any major change is needed.

Reply:
Thanks for clarifying. As long as the service doesn't run on the main docs VM, this is fine.

Reply:
They are independent. I don't think even the CI will fail if the disk becomes full. But as said, planning to implement the purging and anything else needeed before that happens, once we see everythinf is working fine.